core/templates/dev/head/components/answer_group_editor_directive.html

```diff
@@ -50,4 +50,47 @@
                     suppress-warnings="suppressWarnings()">
     </outcome-editor>
   </div>
+  <style>
+    .oppia-rule-header {
```

Comment:
the above is in the format _directive-name.css-class-name_ , right ?
I'll do the changes and let you know. 
Sorry, I didn't look at the comments thinking they were regarding script.